### DISCUSSION TITLE
[@mantine/form] Can the event that is the first argument of the method generated by onSubmit be optional?

### DISCUSSION THREAD

**Discussion by yicru** [mantinedev/mantine#2858] [Q&A] [answered]
I want to fire onSubmit with no arguments to invoke remotely.

[React Hook Form's handleSubmit API](https://react-hook-form.com/api/useform/handlesubmit)
> // It can be invoked remotely as well
> handleSubmit(onSubmit)();

**Comment by rtivital** [comment, marked as answer]
I'll add this feature in one of the next minor releases

**Reply by yicru** [reply]
Nice ! 
Thank you for the reply.
